neuralop/layers/embeddings.py

```diff
@@ -73,4 +74,170 @@ def apply_2d_rotary_pos_emb(t, freqs_x, freqs_y):
         t_x, t_y = t[..., :d//2], t[..., d//2:]
 
         return torch.cat((apply_rotary_pos_emb(t_x, freqs_x),
-                          apply_rotary_pos_emb(t_y, freqs_y)), dim=-1)
\ No newline at end of file
+                          apply_rotary_pos_emb(t_y, freqs_y)), dim=-1)
+
+
+class GaussianFourierFeatureTransform(nn.Module):
```

Comment:
@dhpitt We now have two Fourier feature embeddings, i.e., `PositionalEmbedding` above and this one --- I think we should name them more consistently, e.g., `GaussianFourierEmbedding` and `FourierEmbedding`?

Reply:
Good idea @zijieli-Jlee